### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_AzureSQLMigration_Colorcontrast:The color contrast ratio is 2.4:1 when there is a visible indication of focus on the learn more link.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15730]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#AINotInScope;#Colorcontrast

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. Tab till learn more link.
5. Open color contrast Analyser and check for color contrast when the focus is visible.

### Actual:
The color contrast ratio is 2.4:1 when there is a visible indication of focus on the learn more link.
**This issue is observed with links throughout the application.**

### Expected:
The color contrast ratio should be greater than or equal to 3:1 .

### User Impact:
People with low vision often have difficulty in seeing the focus on the links.

**Comment by aasimkhan30** [comment]
This is a core bug.

**Comment by prabhudevu** [comment]
![15730_Issue not fixed](https://user-images.githubusercontent.com/98821792/216965115-d9e2df4d-c3a5-4929-9cd8-6ba883ddf927.png)

Verified the issue in the latest Build 1.42.0 

Issue still not fixed. The color contrast ratio is 2.396:1 when there is a visible indication of focus on the learn more link.

Please reopen the bug issue is not fixed.

**GithubTags**:#Regressed:02-06-23;

**Comment by Roopesh-T** [comment]
Above issue is not repro in latest version
Version: 1.43.0-insider (user setup)
Commit: 9d16a48dee06c0533c5c6e405e775685d098a5ae
Date: 2023-03-20T05:49:59.560Z
VS Code: 1.67.0
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
![image](https://user-images.githubusercontent.com/86601127/226274898-8f606a25-796f-4f63-bf90-d986f0c0fa15.png)

** GitHub Tags "" : #Closed;
